src/plots/wheelPlot.ts

```diff
@@ -0,0 +1,219 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Plots {
+  export class Wheel<R, T> extends Plot {
+
+    private static _INNER_RADIUS_KEY = "inner-radius";
+    private static _OUTER_RADIUS_KEY = "outer-radius";
+    private static _START_ANGLE_KEY = "start-angle";
+    private static _END_ANGLE_KEY = "end-angle";
+
+    /**
+     * @constructor
+     */
+    constructor() {
+      super();
+      this.addClass("wheel-plot");
+      this.attr("fill", (d, i) => String(i), new Scales.Color());
```

Comment:
I don't think this makes sense for `Plots.Wheel`; consider a parallel with `Plots.Rectangle`, which just has a single-color fallback if the user doesn't set `fill`.